Disable building manpages so that make install doesn't fail due to lack of help2man

Upstream-Status: Inappropriate [config]

Signed-off-by: Paul Eggleton <paul.eggleton@linux.intel.com>

diff --git a/src/Makefile.am b/src/Makefile.am
index 46c4017..d6ed312 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -9,11 +9,6 @@ izcoordinator_DESC = "simple coordinator for IEEE 802.15.4 network"
 iz_DESC = "configure an IEEE 802.15.4 interface"
 izchat_DESC = "simple chat program using IEEE 802.15.4"
 
-if MANPAGES
-dist_man_MANS = $(manpages)
-endif
-EXTRA_DIST = $(manpages)
-
 izattach_SOURCES = serial.c
 
 iz_SOURCES = iz.c iz-common.c iz-mac.c iz-phy.c
@@ -27,18 +22,6 @@ izcoordinator_LDADD = ../addrdb/libaddrdb.la $(LDADD) $(NL_LIBS) $(LEXLIB)
 iz_CFLAGS = $(AM_CFLAGS) $(NL_CFLAGS) -D_GNU_SOURCE
 iz_LDADD = $(LDADD) $(NL_LIBS)
 
-izattach.8: $(izattach_SOURCES) $(top_srcdir)/configure.ac
-	-$(HELP2MAN) -o $@ -s 8 -N -n $(izattach_DESC) $(builddir)/izattach
-
-izcoordinator.8: $(izcoordinator_SOURCES) $(top_srcdir)/configure.ac
-	-$(HELP2MAN) -o $@ -s 8 -N -n $(izcoordinator_DESC) $(builddir)/izcoordinator
-
-iz.8: $(iz_SOURCES) $(top_srcdir)/configure.ac
-	-$(HELP2MAN) -o $@ -s 8 -N -n $(iz_DESC) $(builddir)/iz
-
-izchat.1: $(izchat_SOURCES) $(top_srcdir)/configure.ac
-	-$(HELP2MAN) -o $@ -s 1 -N -n $(izchat_DESC) $(builddir)/izchat
-
 install-data-hook:
 	$(mkinstalldirs) $(DESTDIR)`dirname $(leasefile)`
 	$(mkinstalldirs) $(DESTDIR)`dirname $(pidfile)`
